Patch104pZH/GameFilesEdited/Data/INI/Object/FactionBuilding.ini

```
@@ -7402,6 +7402,786 @@ Object AmericaStrategyCenter
+    AliasConditionState = SOLD
+    AliasConditionState = SOLD DAMAGED
+    AliasConditionState = SOLD REALLYDAMAGED RUBBLE
+  End
+
+  PlacementViewAngle = -45
+
+  ; ***DESIGN parameters ***
+  DisplayName         = OBJECT:DetentionCamp
+  Side                = America
+  EditorSorting       = STRUCTURE
+  Prerequisites
+    Object            = AmericaStrategyCenter
+  End
+  BuildCost           = 1000
+  BuildTime           = 30.0           ; in seconds
+  EnergyProduction    = 0
+  VisionRange         = 200.0           ; Shroud clearing distance
+  ShroudClearingRange = 200
+  ArmorSet
+    Conditions        = None
+    Armor             = StructureArmor
+    DamageFX          = StructureDamageFXNoShake
+  End
+  CommandSet          = AmericaDetentionCampCommandSet
+  ExperienceValue     = 100 100 100 100  ; Experience point value at each level
+
+  ; *** AUDIO Parameters ***
+  VoiceSelect         = DetentionCampSelect
+  SoundOnDamaged        = BuildingDamagedStateLight
+  SoundOnReallyDamaged  = BuildingDestroy
+  SoundEnter = POWTruckEnter
+  SoundExit = POWTruckExit
+
+  UnitSpecificSounds
+    UnderConstruction     = UnderConstructionLoop
+  End
+
+  ; *** ENGINEERING Parameters ***
+  RadarPriority       = STRUCTURE
+  KindOf              = PRELOAD STRUCTURE SELECTABLE IMMOBILE SCORE IMMUNE_TO_CAPTURE FS_TECHNOLOGY MP_COUNT_FOR_VICTORY FS_FACTORY ;AUTO_RALLYPOINT
+  Body                = StructureBody ModuleTag_05
+    MaxHealth         = 2000.0
+    InitialHealth     = 2000.0
+    ; Subdual damage "Subdues" you (reaction defined by BodyModule) when it passes your max health.
+    ; The cap limits how extra-subdued you can be, and the other numbers detemine how fast it drains away on its own.
+    SubdualDamageCap = 2200
+    SubdualDamageHealRate = 500
+    SubdualDamageHealAmount = 100
+  End
+
+  Behavior = DefaultProductionExitUpdate ModuleTag_06
+    UnitCreatePoint = X:-28.0 Y:30.0 Z:0.0
+    NaturalRallyPoint = X:-28.0 Y:49.0 Z:0.0
+  End
+
+  Behavior = ProductionUpdate ModuleTag_07
+    NumDoorAnimations = 1
+    DoorOpeningTime = 3250
+    DoorWaitOpenTime = 3000
+    DoorCloseTime = 4000
+    ConstructionCompleteDuration = 1500
+  End
+
+  Behavior                   = SpyVisionSpecialPower SpyTag_1
+    SpecialPowerTemplate     = SuperweaponCIAIntelligence
+    BaseDuration             =  30000 ;in milliseconds
+    BonusDurationPerCaptured =  10000 ;in milliseconds
+    MaxDuration              = 240000 ;in milliseconds
+  End
+  Behavior                   = SpyVisionUpdate SpyTag_2
+    ;<NO DATA>
+  End
+
+  Behavior            = BaseRegenerateUpdate ModuleTag_08
+    ;<NO DATA>
+  End
+  Behavior            = DestroyDie ModuleTag_10
+    ;<NO DATA>
+  End
+  Behavior            = CreateObjectDie ModuleTag_12
+    CreationList      = OCL_ABPowerPlantExplode
+  End
+
+  Behavior            = CreateObjectDie ModuleTag_13
+    CreationList      = OCL_AmericanRangerDebris04
```

```suggestion
    CreationList      = OCL_AmericanRangerDebris03
```
Interestingly, it does spawn 8 in CCG while costing $1000: https://github.com/TheSuperHackers/GeneralsGamePatch/blob/main/Patch104pZH/GameFilesOriginalCCG/Data/INI/Object/FactionBuilding.ini#L5657-L5660